app/src/main/java/org/mozilla/fenix/compose/Image.kt

```diff
@@ -46,25 +46,29 @@ fun Image(
     alignment: Alignment = Alignment.Center,
     contentScale: ContentScale = ContentScale.Fit,
 ) {
-    ImageLoader(
-        url = url,
-        client = components.core.client,
-        private = private,
-        targetSize = targetSize
-    ) {
-        WithImage { painter ->
-            androidx.compose.foundation.Image(
-                painter = painter,
-                modifier = modifier,
-                contentDescription = contentDescription,
-                alignment = alignment,
-                contentScale = contentScale
-            )
-        }
+    if (inComposePreview) {
+        DefaultImagePlaceholder(modifier = modifier)
```

Comment:
Can you also go into DefaultImagePlaceholder and use an appropriate color variable for the color in there.

Reply:
sure, what are we thinking, layer 2?

Reply:
Whatever was closest to the current colors was good enough with me. Didn't have anything specific in mind